backend/api/v1/endpoints/auth_email.py

```
@@ -0,0 +1,78 @@
+import os
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)